SearchBayesianSkopt: Resuming 'SLIM_BPR_Recommender' Failed, no such file exists.
SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 713, 'epochs': 1500, 'symmetric': True, 'sgd_mode': 'adam', 'lambda_i': 0.0023912227215105372, 'lambda_j': 0.0003661497827181699, 'learning_rate': 0.014238198417759912} - Exception: Traceback (most recent call last):
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 468, in _objective_function
    result_df, recommender_instance = self._evaluate_on_validation(current_fit_hyperparameters_dict, was_already_evaluated_flag, was_already_evaluated_index)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 326, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_hyperparameters)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 297, in _fit_model
    recommender_instance.fit(*self.recommender_input_args.FIT_POSITIONAL_ARGS,
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/Recommenders/SLIM/Cython/SLIM_BPR_Cython.py", line 76, in fit
    from Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch import SLIM_BPR_Cython_Epoch
ModuleNotFoundError: No module named 'Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch'

SearchBayesianSkopt: Resuming 'SLIM_BPR_Recommender' Failed, no such file exists.
SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 949, 'epochs': 1500, 'symmetric': True, 'sgd_mode': 'adam', 'lambda_i': 2.215863787312883e-05, 'lambda_j': 0.00670000321573401, 'learning_rate': 0.00263734164338454} - Exception: Traceback (most recent call last):
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 468, in _objective_function
    result_df, recommender_instance = self._evaluate_on_validation(current_fit_hyperparameters_dict, was_already_evaluated_flag, was_already_evaluated_index)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 326, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_hyperparameters)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 297, in _fit_model
    recommender_instance.fit(*self.recommender_input_args.FIT_POSITIONAL_ARGS,
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/Recommenders/SLIM/Cython/SLIM_BPR_Cython.py", line 76, in fit
    from Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch import SLIM_BPR_Cython_Epoch
ModuleNotFoundError: No module named 'Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch'

SearchBayesianSkopt: Config 1 Exception. Config: {'topK': 533, 'epochs': 1500, 'symmetric': True, 'sgd_mode': 'sgd', 'lambda_i': 0.0010192242539579898, 'lambda_j': 0.00014442658630862368, 'learning_rate': 0.00012736348485694454} - Exception: Traceback (most recent call last):
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 468, in _objective_function
    result_df, recommender_instance = self._evaluate_on_validation(current_fit_hyperparameters_dict, was_already_evaluated_flag, was_already_evaluated_index)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 326, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_hyperparameters)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 297, in _fit_model
    recommender_instance.fit(*self.recommender_input_args.FIT_POSITIONAL_ARGS,
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/Recommenders/SLIM/Cython/SLIM_BPR_Cython.py", line 76, in fit
    from Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch import SLIM_BPR_Cython_Epoch
ModuleNotFoundError: No module named 'Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch'

SearchBayesianSkopt: Config 2 Exception. Config: {'topK': 136, 'epochs': 1500, 'symmetric': False, 'sgd_mode': 'adagrad', 'lambda_i': 0.0012893954495430036, 'lambda_j': 2.3484178489933954e-05, 'learning_rate': 0.02975480431993266} - Exception: Traceback (most recent call last):
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 468, in _objective_function
    result_df, recommender_instance = self._evaluate_on_validation(current_fit_hyperparameters_dict, was_already_evaluated_flag, was_already_evaluated_index)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 326, in _evaluate_on_validation
    recommender_instance, train_time = self._fit_model(current_fit_hyperparameters)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/HyperparameterTuning/SearchAbstractClass.py", line 297, in _fit_model
    recommender_instance.fit(*self.recommender_input_args.FIT_POSITIONAL_ARGS,
  File "/Users/jodyrobertobattistini/PycharmProjects/rec-sys-challenge/Recommenders/SLIM/Cython/SLIM_BPR_Cython.py", line 76, in fit
    from Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch import SLIM_BPR_Cython_Epoch
ModuleNotFoundError: No module named 'Recommenders.SLIM.Cython.SLIM_BPR_Cython_Epoch'

SearchBayesianSkopt: Search interrupted. No valid config was found during the initial random initialization
